pkg/manager/impl/shared/constants.go

```diff
@@ -33,6 +33,9 @@ const (
 	Attributes            = "attributes"
 	MatchingAttributes    = "matching_attributes"
 	// Parent of a node execution in the node executions table
-	ParentID        = "parent_id"
-	WorkflowClosure = "workflow_closure"
+	ParentID                = "parent_id"
+	WorkflowClosure         = "workflow_closure"
+	BaseExecutionIDLabelKey = "base_exec_id"
```

Comment:
Should we call it `parent_exec_id`? just to keep names consistent... I understand that for "self" execution that will sound weird... 🤷🏽 or maybe keep them as two separate labels?